lib/src/view_version.dart

```diff
@@ -278,16 +256,15 @@ class ViewVersion {
     }
     row.data = cells;
   }
-
-  static _typeCell(String declaredType, String inferredType, {colspan: '1'}) {
-    return _verticalCell(new SpanElement()
-      ..appendText('inferred: ')
-      ..append(_span(inferredType, cssClass: 'preSpan')), new SpanElement()
-      ..appendText('declared: ')
-      ..append(_span(declaredType, cssClass: 'preSpan')), colspan: colspan);
-  }
 }
 
+TableCellElement _typeCell(String declaredType, String inferredType,
+    {colspan: '1'}) => _verticalCell(new SpanElement()
```

Comment:
I don't necessarily agree with use of the shorthand function syntax here, but its up to you.